### ISSUE TITLE
task: allow viewing all tasks that a user has access to

### ISSUE THREAD

**Issue by mark-rushakoff** [influxdata/platform#997]
There are currently two ways to filter viewing tasks: by user ID and by organization. Neither of these give the whole picture of what tasks a user has access to. That would require identifying every organization a user belongs to - maybe that ties in with the membership work?

This is of interest to the task UI.

**Comment by goller** [comment]
@mark-rushakoff yup

**Comment by mark-rushakoff** [comment]
Tentatively, we're going to depend on a Read tasks from an organization Authorization to know which orgs to look up.

**Comment by mark-rushakoff** [comment]
> I'd like to drop organization and owner references from tasks

Does this mean the task storage layer wouldn't be concerned about the bookkeeping of which tasks belong to which user/org? That sounds great.

I guess there would be some task internal API changes to deal with that, but it would most likely all be simpler than the current code to manage user and org bookkeeping.

**Comment by jademcgough** [comment]
> Does this mean the task storage layer wouldn't be concerned about the bookkeeping of which tasks belong to which user/org? That sounds great.

Yep! I think it makes sense to keep that out of the concerns of the task storage layer. Resource ownership should be decoupled from the resources themselves, and it's something that needs to be handled (in probably an identical manner) for all user or org owned resources.

**Comment by mark-rushakoff** [comment]
Looks like this is still blocked on #1019 and #1020?
